packages/wp-now/src/pool.ts

```diff
@@ -0,0 +1,261 @@
+import { NodePHP } from '@php-wasm/node';
+
+let childCount = 0;
+
+/**
+ * PRIVATE
+ * Tracks stats of instances in a pool.
+ */
+class PoolInfo {
+	id = childCount++; // Unique ID for debugging purposes.
+	requests = 0; // Total requests processed.
+	started = Date.now(); // Time spawned.
+	active = false; // Whether instance is considered active.
+}
+
+/**
+ * PRIVATE
+ * Spawns new instances if the pool is not full.
+ * Returns a list of new instances.
+ */
+const spawn = (pool) => {
+	const newInstances = new Set();
+
+	while (pool.maxJobs > 0 && pool.instanceInfo.size < pool.maxJobs) {
```

Comment:
Nitpick: pool.maxJobs doesn't change in the loop and makes the condition a bit more difficult to follow. How about`if ( pool.maxJobs <= 0 ) { return newInstances; }` before the loop?

Reply:
@adamziel That would definitely be clearer. I'll go ahead and make the change.